03 Writing Algorithms/34 Algorithm Framework/07 Hybrid Algorithms/06 Execution.html

```diff
@@ -1,7 +1,13 @@
 <p>
-  <a href="/docs/v2/writing-algorithms/algorithm-framework/execution/supported-models">Execution models</a> can place orders for your strategy instead of plancing them manually. To do this route PortfolioTarget requests to the ExecutionModel...
+  <a href="/docs/v2/writing-algorithms/algorithm-framework/execution/supported-models">Execution models</a> can place orders for your strategy instead of placing them manually. To do so,
+  <a href="/docs/v2/writing-algorithms/algorithm-framework/execution/key-concepts#05-Portfolio-Target-Collection">PortfolioTarget</a>s are routed to the <code>Execute</code> method of the Execution Model to place orders.
 </p>
 <p>See the below strategy for an example using a Framework Execution Model in a classic style algorithm:</p>
 <div class="section-example-container">
-     <pre class="Csharp"></pre><pre class="python"></pre>
-</div>
+	<pre class="csharp">public override void Initialize()
+{
+    SetExecution(new ImmediateExecutionModel());
```

Comment:
The `ImmediateExecutionModel` is the default model in `QCAlgorithm` (framework or classic), so we could propose a different model here.